Recipe-App/Recipe-App/UIViews/Components/RecipeCardItemView.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  ImageTitleRatingCardView.swift
+//  Recipe-App
+//
+//  Created by Gontse Ranoto on 2021/06/14.
+//
+
+import SwiftUI
+
+struct RecipeCardItemView: View {
+    @Binding var imageRecipeName: String
```

Comment:
Can we just add a normal `var recipe: Recipe` objects instead of using bindings because we not going to update any data we also don't have to pass multiple parameters.

You can add this dummy data to your constant
`   static let dummyRecipeList: [Recipe] = [
    Recipe(id: 1, title: "Chicken soup with mayo", publisher: nil, featuredImage: "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcReU_iHN3DdMKVhOgtJgQW6qmYxPCFZNoenWw&usqp=CAU", rating: 80, sourceUrl: nil, description: nil, cookingInstructions: nil, ingredients: nil, dateAdded: nil, dateUpdated: nil),
                                           Recipe(id: 1, title: "Chicken noodle soup", publisher: nil, featuredImage: "https://cafedelites.com/wp-content/uploads/2018/03/Chicken-Noodle-Soup-IMAGE-37.jpg", rating: 80, sourceUrl: nil, description: nil, cookingInstructions: nil, ingredients: nil, dateAdded: nil, dateUpdated: nil),
                                            Recipe(id: 1, title: "Chicken pasta soup", publisher: nil, featuredImage: "https://www.eatwell101.com/wp-content/uploads/2020/02/chicken-soup-recipe-3.jpg", rating: 70, sourceUrl: nil, description: nil, cookingInstructions: nil, ingredients: nil, dateAdded: nil, dateUpdated: nil)]`